pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/MinionClusterIntegrationTest.java

```diff
@@ -18,76 +18,58 @@
 import com.google.common.util.concurrent.Uninterruptibles;
 import com.linkedin.pinot.common.config.AbstractTableConfig;
 import com.linkedin.pinot.common.config.PinotTaskConfig;
-import com.linkedin.pinot.common.utils.CommonConstants;
-import com.linkedin.pinot.common.utils.ZkStarter;
 import com.linkedin.pinot.controller.helix.core.PinotHelixResourceManager;
 import com.linkedin.pinot.controller.helix.core.minion.ClusterInfoProvider;
 import com.linkedin.pinot.controller.helix.core.minion.PinotHelixTaskResourceManager;
 import com.linkedin.pinot.controller.helix.core.minion.PinotTaskManager;
 import com.linkedin.pinot.controller.helix.core.minion.generator.PinotTaskGenerator;
-import com.linkedin.pinot.minion.MinionStarter;
 import com.linkedin.pinot.minion.exception.TaskCancelledException;
 import com.linkedin.pinot.minion.executor.BaseTaskExecutor;
+import com.linkedin.pinot.minion.executor.PinotTaskExecutor;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import javax.annotation.Nonnull;
-import org.apache.commons.configuration.Configuration;
-import org.apache.commons.configuration.PropertiesConfiguration;
-import org.apache.helix.task.TaskDriver;
 import org.apache.helix.task.TaskState;
 import org.testng.Assert;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 
-public class MinionClusterIntegrationTest {
-  private static final int ZK_PORT = 2191;
-  private static final String ZK_ADDRESS = "localhost:" + ZK_PORT;
-  private static final String HELIX_CLUSTER_NAME = "pinot-minion-test";
-  private static final String CONTROLLER_ID = "controller";
+public class MinionClusterIntegrationTest extends HybridClusterIntegrationTest {
+  private static final String RAW_TABLE_NAME = "mytable";
+  private static final String OFFLINE_TABLE_NAME = "mytable_OFFLINE";
+  private static final String REALTIME_TABLE_NAME = "mytable_REALTIME";
   private static final int NUM_WORKERS = 3;
 
-  private ZkStarter.ZookeeperInstance _zookeeperInstance;
   private PinotHelixResourceManager _pinotHelixResourceManager;
   private PinotHelixTaskResourceManager _pinotHelixTaskResourceManager;
   private PinotTaskManager _pinotTaskManager;
-  private MinionStarter[] _minionStarters = new MinionStarter[NUM_WORKERS];
 
   @BeforeClass
   public void setUp()
       throws Exception {
-    // Start a ZooKeeper instance.
-    _zookeeperInstance = ZkStarter.startLocalZkServer(ZK_PORT);
+    super.setUp();
```

Comment:
Add a comment that the parent `setUp` sets up ZK, kafka, controller, brokers and servers.